simple_audit/signal.py

```diff
@@ -236,9 +234,9 @@ def save_audit(instance, operation, kwargs={}):
                         format_value(v[1]),
                     ) for k, v in changed_fields.items()])
         elif operation == Audit.DELETE:
-            description = _('Deleted %s') % unicode(instance)
+            description = _('Deleted %s') % str(instance)
```

Comment:
`str` on python2 does very different things than `unicode`. Should we bring in six and use `six.text_type`?

Reply:
I don't think that's necessary in this case. `str` and `unicode` definitely behave differently, but with the `unicode_literals` import at the top of this file it should bridge the gap where needed. Also, this is just storing a string in the db logging what changed and how it changed. 

This is what I was using for reference [link](http://python-future.org/compatible_idioms.html#strings-and-bytes)

Reply:
`unicode_literals` stops you from having to type the `u` on `u'my hardcoded string'`.

Reply:
Ah, good catch. I can bring in `six.text_type` and replace the instances of `str` with that.